README.md

```diff
@@ -7,8 +7,7 @@ Command-line tool for finding Bazel dependencies which are not actually used in
 ## What is an unused dependency?
 
 An unused Bazel dependency is an unnecessary entry in the `deps` field of a `java_library` or `java_binary` target in
-a `BUILD` file, meaning the code being compiled doesn't actually have any dependency on that library. See the Appendix
-for an example! Over time, unused dependencies accumulate as code changes and `BUILD` files are not updated to reflect
+a `BUILD` file, meaning the code being compiled doesn't actually have any dependency on that library. Over time, unused dependencies accumulate as code changes and `BUILD` files are not updated to reflect
```

Comment:
This line is really out of line with the width of the rest of the document. This may throw off anyone reading this not on github (or a pretty-printed format).